src/formBuilder/utils.js

```diff
@@ -355,14 +355,15 @@ export function checkForUnsupportedFeatures(
 }
 
 // make an element out of the corresponding properties and UI properties
-function generateElement(
+function generateDependencyElement(
   name: string,
   dataProps: any,
   uiProperties: any,
   requiredNames: any,
   definitionData?: { [string]: any },
   definitionUi?: { [string]: any },
   categoryHash: { [string]: string },
+  useReferenceDetails: boolean = true, //determines whether to use reference details or not.
```

Comment:
```suggestion
  useReferenceDetails: boolean = true, // determines whether to use reference details or not.
```

I'm not sure I understand by reading the comment what it does.  What is a "reference detail" exactly?

Reply:
Reference details being the definition details. I will update, thanks.